README.md

```diff
@@ -1,25 +1,68 @@
+<p align="center">
+    <p align="center">
+        git + <img src="https://user-images.githubusercontent.com/3167497/34473826-40b4987c-ef2c-11e7-90b9-5ff322c4966f.png" width="100" height="100"> = gitlab
+    </p>
+    <p align="center">
+        <a href="https://travis-ci.org/zaquestion/lab">
+            <img src="https://travis-ci.org/zaquestion/lab.svg?branch=master" alt="Build Status">
+        </a>
+        <a href="https://goreportcard.com/report/github.com/zaquestion/lab">
+            <img src="https://goreportcard.com/badge/github.com/zaquestion/lab" alt="Go Report Card">
+        </a>
+        <a href="https://codecov.io/gh/zaquestion/lab">
+            <img src="https://codecov.io/gh/zaquestion/lab/branch/master/graph/badge.svg" alt="codecov">
+        </a>
+        <a href="https://gitter.im/labcli">
+            <img src="https://badges.gitter.im/Join%20Chat.svg" alt="Join the chat">
+        </a>
+    </p>
+    <p align="center">
+        <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">
+            <img src="http://i.creativecommons.org/p/zero/1.0/88x31.png" alt="CC0 License">
+        </a>
+        <a href="https://liberapay.com/zaquestion/donate">
+            <img src="https://liberapay.com/assets/widgets/donate.svg" alt="Donate">
+        </a>
+    </p>
+    <p align="center">
+        <img src="https://user-images.githubusercontent.com/1964720/42740177-6478d834-8858-11e8-9667-97f193ecb404.gif" align="center">
+    </p>
+</p>
 
-<p align="center"><img src="https://user-images.githubusercontent.com/1964720/42740177-6478d834-8858-11e8-9667-97f193ecb404.gif" align="center"></p>
+_lab_ interact with repositories on GitLab, including creating/editing merge requests, issues, milestones, snippets and
+CI pipelines.
 
-Lab wraps Git, making it simple to clone, fork, and interact with repositories on GitLab, including seamless workflows for creating merge requests, issues and snippets.
+The development team has the focus on keeping _lab_ a simple, the less opinionated possible, command line interface for
+what the GitLab API offers. Also, _lab_ aims to feel familiar to a Git user and follow Git's behaviors/patterns for
+some of its commands and flags.
 
```

Comment:
The development team has focused on keeping _lab_ a simple and intuitive command line interface for commands provided in the GitLab API (link?) .  _lab_'s aim is to provide GitLab users an experience similar to the GitLab webUI with respect to errors and messages.

Reply:
Yes, better :))